content/source/docs/enterprise/api/modules.html.md

```diff
@@ -311,20 +372,33 @@ After the registry module version is successfully parsed by TFE, its status will
 
 ## Delete a Module
 
-These endpoints can delete a single version for a provider, a single provider (and all its versions) for a module, or a whole module. If the requested deletion would leave a provider with no versions or a module with no providers, the empty items will be automatically deleted as well.
+When removing modules, there are three versions of the endpoint, depending on how many parameters are specified.
+
+* If all parameters (module, provider, and version) are specified, the provided version for the given provider of the module is deleted.
+* If module and provider are specified, the given provider for the module is deleted along with all its versions.
+* If only module is specified, the entire module is deleted.
+
+If a version deletion would leave a provider with no versions, the provider will be deleted. If a provider deletion would leave a module with no providers, the module will be deleted.
```

Comment:
One other thing: all of these endpoints currently have the paragraph of explanation first, but the URL path should go first, and the explanation should go after the table of URL parameters.